site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -295,6 +295,17 @@ the steps in [Viewing Warnings](#view_warnings).
       <td>Grants the extension access to the <a href="/docs/extensions/reference/proxy">chrome.proxy</a> API.</td>
       <td><strong>Read and change all your data on all websites</strong></td>
     </tr>
+    <tr id="sessionshistory">
+      <td><code>"sessions"</code> and <code>"history"</code></td>
+      <td>Grants the extension access to the <a href="/docs/extensions/reference/sessions">chrome.sessions</a> API and <a href="/docs/extensions/reference/history">chrome.history</a> API.</td>
```

Comment:
This comment is not blocking. 

If I were writing this whole document from scratch, I'd begin each item with 'Grants access'. It seems safe to assume the reader would know we're talking about extensions. Reducing the repetition would make scanning quicker.